cockroachcloud/managing-cmek.md

```diff
@@ -5,11 +5,11 @@ toc: true
 docs_area: manage.security
 ---
 
-Customer-Managed Encryption Keys (CMEK) for {{ site.data.products.dedicated }} allows the customer to delegate responsibility for the work of encrypting their cluster data to {{ site.data.products.dedicated }}, while maintaining the ability to completely revoke {{ site.data.products.dedicated }}'s access.
+[Customer-Managed Encryption Keys (CMEK)](cmek.html) for {{ site.data.products.dedicated }} allow the customer to delegate responsibility for the work of encrypting their cluster data to {{ site.data.products.dedicated }}, while maintaining the ability to completely revoke {{ site.data.products.dedicated }}'s access.
```

Comment:
I prefer the previous version where we are talking about CMEK the feature rather than CMEK the key.